Remotion/Web/Core/UI/Controls/DropDownMenuImplementation/Rendering/DropDownMenuRenderer.cs

```diff
@@ -71,7 +71,10 @@ public void RegisterHtmlHeadContents (HtmlHeadAppender htmlHeadAppender)
       ArgumentUtility.CheckNotNull ("htmlHeadAppender", htmlHeadAppender);
 
       htmlHeadAppender.RegisterUtilitiesJavaScriptInclude();
-      htmlHeadAppender.RegisterJQueryIFrameShimJavaScriptInclude();
+
+      string jQueryIframeShimScriptKey = typeof (HtmlHeadContents).FullName + "_JQueryBgiFrames";
+      var href = ResourceUrlFactory.CreateResourceUrl (typeof (HtmlHeadContents), ResourceType.Html, "jquery.IFrameShim.js");
+      htmlHeadAppender.RegisterJavaScriptInclude (jQueryIframeShimScriptKey, href);
```

Comment:
Something went wrong in the review comment. I meant we should inline the relevant code into the `RegisterUtilitiesJavaScriptInclude` method. It's going to get dropped anyway when we delte the IE code later this release.

Reply:
What relevant code? I still don't follow

Reply:
These three lines of code. I didn't mean to inline them 3 times in various files, just once into the `RegisterUtilitiesJavaScriptInclude` method.